Pipfile

```diff
@@ -4,10 +4,11 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-sceptre = "2.4.0"
```

Comment:
Please ignore changes to Pipfile and Pipfile.lock -- these changes are being made separately in #19 .